modules/videoio/src/cap_ffmpeg_impl.hpp

```diff
@@ -1591,6 +1591,9 @@ double CvCapture_FFMPEG::getProperty( int property_id ) const
     case CAP_PROP_HW_ACCELERATION_USE_OPENCL:
         return static_cast<double>(use_opencl);
 #endif  // USE_AV_HW_CODECS
+    case CAP_PROP_STREAM_OPEN_TIME_USEC:
+        //ic->start_time_realtime is in microseconds
+        return ((double)ic->start_time_realtime); 
```

Comment:
Please fix whitespace issue: http://pullrequest.opencv.org/buildbot/builders/precommit_docs/builds/31979

Reply:
`git diff --check` did not catch this - is that a bug? I'm assuming that the extra space at the end of line 1595 was the offending whitespace.

Reply:
It works (need to specify the "base" commit):

```
$ git checkout 567ffd8134d67d76277181930dd4f4eb618fe26f
Note: switching to '567ffd8134d67d76277181930dd4f4eb618fe26f'.

HEAD is now at 567ffd8134 use microseconds for CAP_PROP_STREAM_OPEN_TIME (nanoseconds rolls over too soon, and milliseconds/seconds requires a division)
$ git diff --check 6fbfc58602e719b9d58099a3c43e5c764718c4ba
modules/videoio/src/cap_ffmpeg_impl.hpp:1596: trailing whitespace.
+        return ((double)ic->start_time_realtime); 
```

Reply:
> It works (need to specify the "base" commit):
> 
> ```
> $ git checkout 567ffd8134d67d76277181930dd4f4eb618fe26f
> Note: switching to '567ffd8134d67d76277181930dd4f4eb618fe26f'.
> 
> HEAD is now at 567ffd8134 use microseconds for CAP_PROP_STREAM_OPEN_TIME (nanoseconds rolls over too soon, and milliseconds/seconds requires a division)
> $ git diff --check 6fbfc58602e719b9d58099a3c43e5c764718c4ba
> modules/videoio/src/cap_ffmpeg_impl.hpp:1596: trailing whitespace.
> +        return ((double)ic->start_time_realtime); 
> ```
Sorry - do I need to do something on this PR, or are you saying that, in order to get the whitespace check to work, I need to use `git diff --check [base_commit]` ?

Reply:
> do I need to do something on this PR

No. This PR is merged. Thank you for contribution!

See here about automatic local whitespace checks: https://github.com/opencv/opencv/wiki/How_to_contribute#q3-i-was-asked-to-remove-whitespace-issues-how-can-i-do-that